### ISSUE TITLE
examples missing

### ISSUE THREAD

**Issue by denysvitali** [oauthjs/node-oauth2-server#287]
As stated in the title, the directory "examples" is missing, thus the linked examples in the README.md don't work

**Comment by julestruong** [comment]
examples seem to work if you switch to 2.4.1 branch.

**Comment by richardrissanen** [comment]
The examples directory appears to have been removed in [this commit](https://github.com/oauthjs/node-oauth2-server/commit/6b182e146be9114eb3d0fdfa5fc0ee87dc61e552). However, it did not appear to contain the linked examples that are missing within the README.md. I also do not see this 2.4.1 branch that [julestruong](https://github.com/julestruong) has mentioned. If anyone would like to provide a link that would be helpful.

**Comment by sebilasse** [comment]
@richardrissanen How about the express wrapper: https://github.com/oauthjs/express-oauth-server –
but it seems the examples are not 3.x : https://github.com/oauthjs/express-oauth-server/issues/10

**Comment by julestruong** [comment]
@richardrissanen  i edited

**Comment by richardrissanen** [comment]
@julestruong I appreciate the edit. I believe there is a mismatch from the examples directory and what I believe @denysvitali was referencing. In the reademe.md there is a section called examples and these three bullet points are the ones that I am referring to and I believe @denysvitali is referring to.

  - A simple **password** grant authorization [example](examples/password).
  - A more complex **password** and **refresh_token** [example](examples/refresh-token).
  - An advanced **password**, **refresh_token** and **authorization_code** (with scopes) [example](examples/authorization-code).

The bullet points mentioned have links that when referencing their source do not appear to be fixed by the examples directory in v2.2.1.

**Comment by denysvitali** [comment]
Exactly, I was referencing to the bullet point examples. They return a 
404 because the examples/ folder is missing

**Comment by richardrissanen** [comment]
@denysvitali Sweet! @everyone For added emphasis when comparing the bullet points source to the examples directory in v2.2.1 it does not appear to have what is needed in order for them to work.

Here is the source from the links:

https://github.com/oauthjs/node-oauth2-server/issues/examples/password
https://github.com/oauthjs/node-oauth2-server/issues/examples/refresh-token
https://github.com/oauthjs/node-oauth2-server/issues/examples/authorization-code

And v2.2.1 has an examples directory but it is filled with other directories and no reference to `/password`, `/refresh-token`, or `/authorization-code` to my knowledge.

**Comment by richardrissanen** [comment]
The mystery deepens. This is where the examples were added to the readme.md [examples added to read.md](https://github.com/oauthjs/node-oauth2-server/commit/0c50f976d5770a4a2cf12aaf45df9fe5aa934ed9#diff-04c6e90faac2675aa89e2176d2eec7d8R70). If you browse the tree at that time there was still no examples directory. [no examples at the time](https://github.com/oauthjs/node-oauth2-server/tree/0c50f976d5770a4a2cf12aaf45df9fe5aa934ed9). I am now uncertain if these examples have ever worked.

**Comment by manjeshpv** [comment]
Full Implementation https://github.com/manjeshpv/node-oauth2-server-implementation

**Comment by maxtruxa** [comment]
For future reference:

Examples for the stable version currently published on npm ([`oauth2-server@2.4.1`](https://www.npmjs.com/package/oauth2-server)) are available in this commit: [b36a06b445ad0a676e6175d68a8bd0b2f3353dbf](https://github.com/oauthjs/node-oauth2-server/tree/b36a06b445ad0a676e6175d68a8bd0b2f3353dbf).

There are currently no (official) examples for v3.
